CyphalRobotController07-CAN-firmware.ino

```diff
@@ -549,6 +557,21 @@ void loop()
     ina226_current_total_mAh += ina226_current_mA/3600.0;
     ina226_power_total_mWh += ina226_power_mW/3600.0;
   }
+  /* check motor timeout */
+  if(timeout_ms_motor0<0xFFFF)
```

Comment:
Drop that check along with having this as a parameter. Make it one second, end of story.

Reply:
nope, i can't agree on that.
The controller should be somehow generic. And there might be robots where 1 second is OK. But there might be others where 1 second is too long. And for test purposes I don't want any timeout at all.
Ant that's why it is configurable. And I think the additional code for having it configurable isn't that much.

Reply:
If you continue to maintain this code ... :stuck_out_tongue_closed_eyes: :wink: